### PR TITLE
Update CPropertyValue.php

### PR DESCRIPTION
**Pull request by devcommiter**
Fixed an error in the eval() usage in the ensureArray method.

### CODE REVIEW

**Comment by samdark** [comment]
Why closing?

**Comment by devcommiter** [comment]
Oh, it's my fail, sorry. I'll re-create requests.

File: framework/utils/CPropertyValue.php [hunk 1]
```diff
@@ -113,8 +113,14 @@ public static function ensureArray($value)
 			$len = strlen($value);
 			if ($len >= 2 && $value[0] == '(' && $value[$len-1] == ')')
 			{
-				eval('$array=array'.$value.';');
-				return $array;
+				try
+				{
+					return @eval('return array' . $value . ';');
```
Comment (cebe) [inline framework/utils/CPropertyValue.php]: why did you add the `@` here?

Comment (devcommiter) [inline framework/utils/CPropertyValue.php]: For suppressing errors from eval() function. I think in this case we will simply parse a necessary exception or get clear false without any additional messages. Similar solution you can see \CChoiceFormat::evaluate or in the \CAuthManager::executeBizRule or \CTimestampBehavior::getTimestampByAttribute

Comment (samdark) [inline framework/utils/CPropertyValue.php]: That changes behavior to be silent if PHP version is lower than 7. I don't think we should change it.

**Comment by samdark** [comment]
Looks good to me now. Please add a like for changelog.

@cebe could you take a brief look again?

**Comment by devcommiter** [comment]
Please, do not spend time for checking this request, I'd better add more changes around eval() into 'patch-2'(https://github.com/yiisoft/yii/pull/4130) and this change for CPropertyValue.php also.
(p.s. I don't want to add many lines into the changelog with the same meaning). What do you think?

**Comment by samdark** [comment]
OK.
